covquestions-js/src/questionnaireEngine.ts

```diff
@@ -361,9 +361,12 @@ export class QuestionnaireEngine {
 
   private setAdditionalJsonLogicOperators() {
     function convertToDateString(
-      timestamp: LogicExpression,
-      dateFormat: LogicExpression
-    ) {
+      timestamp?: LogicExpression,
+      dateFormat?: LogicExpression
+    ): string {
+      if (!timestamp || !dateFormat) {
```

Comment:
@ChristophWersal shouldn't it return `null`? because otherwise the tag will be in the qrcode. instead of leaving it out?

Reply:
Good point. Do you know how it was up to now? Probably, we should not change the current behavior.

Reply:
they are left out of the current qr code

Reply:
i will change it.

Reply:
#180